client/js/verification-portal/community-health-center/index.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react'
+import { connect } from 'react-redux'
+import CommunityHealtCenterForm from './form'
+import { chcInputResource } from './chc-input-resource'
```

Comment:
What do you think about calling this `chcFormFields` to make it more clear what is in the file?